python/setup.py

```diff
@@ -68,7 +73,7 @@
         "pandas",
         "Pillow",
         "pyarrow>=6.0",
-        "pyspark==3.1.2",
+        f"pyspark=={spark_version}",
```

Comment:
When we do a rikai release, what version should we package? Do we need to release two versions of rikai , each for one spark version?

if the python code base can work with both spark 3.1 and 3.2, should we just make this `pyspark>=3.1,<3.3`?

Reply:
One more questions, is the scala code `enableIf` built by compile time or runtime?
So if user install rikai, do they need to specify different jar version ?

Reply:
`enableIf` works in compile-time. I'm afraid we have to release `rikai_spark31_2.12`, `rikai_spark32_2.12`, `rikai_spark32_2.13` separately, that's the pain of building a system on top of another big system.